### PR TITLE
API does not return JSON systematically - fix #10

### PR DESCRIPTION
**Pull request by ajoah**

### CODE REVIEW

File: src/UcrmPluginSdk/Service/UcrmApi.php [hunk 1]
```diff
@@ -105,12 +105,12 @@ public static function create(?string $pluginRootPath = null): self
      *
      * @param mixed[] $query
      *
-     * @return mixed[]
+     * @return mixed
```
Comment (keksa) [inline src/UcrmPluginSdk/Service/UcrmApi.php]:
```suggestion
     * @return array|string
```

File: src/UcrmPluginSdk/Service/UcrmApi.php [hunk 2]
```diff
@@ -119,8 +119,12 @@ public function get(string $endpoint, array $query = []): array
                 'query' => $query,
             ]
         );
+        
+        if(current($response->getHeader('Content-Type')) === 'application/json'){
```
Comment (keksa) [inline src/UcrmPluginSdk/Service/UcrmApi.php]:
```suggestion
        if ($response->getHeaderLine('content-type') === 'application/json') {
```

**Review by keksa — CHANGES_REQUESTED** [review]

**Comment by keksa** [comment]
Also you have to target to original repository with the PR, this way it points only to your fork. :)

**Comment by ajoah** [comment]
@keksa Oops sorry. What did I do?  :laughing: So I made a new PR with your suggestions : https://github.com/Ubiquiti-App/UCRM-Plugin-SDK/pull/11

**Pull request closed without merging**